cmd/exchange_template/exchange_template.go

```diff
@@ -177,8 +176,7 @@ func makeExchange(exchangeDirectory string, configTestFile *config.Config, exch
 		},
 		{
 			Name:         "main",
-			Filename:     "main_file.tmpl",
-			FilePostfix:  ".go",
+			Filename:     "rest.go",
 			TemplateFile: "main_file.tmpl",
```

Comment:
🚧 Inconsistency introduced, I think it's all `rest` here
It's not clear why the `TemplateFile`s have the suffix `_file` in them.
I suggest removing it from all of these.
```suggestion
			Name:         "rest",
			Filename:     "rest.go",
			TemplateFile: "rest.tmpl",
```

Reply:
This does make sense!

Reply:
Sorry, not addressed. I'll re-raise it on the new changes to avoid confusion.